tests/php/sync/test_class.jetpack-sync-plugins.php

```diff
@@ -92,6 +92,11 @@ public function test_edit_plugin() {
 		);
 		set_current_screen( 'plugin-editor' );
 
+		/**
+		 * Used to signal that a plugin was updated
+		 *
+		 * @since 2.6.0
+		 */
```

Comment:
It should say something like 
/** This action is already documented in filename.php as a dynamic action ....  */